docs/content/documentation/drivers.md

```diff
@@ -360,7 +360,7 @@ These additions are enough to make your driver resistant to race conditions in v
 
 ## Extensibility
 
-Cycle.js is a very small framework, and Cycle *DOM*'s Driver is available as an optional plugin for your app. This means it is simple to replace the DOM Driver with any other driver function providing interaction with the user.
+Cycle.js is a very small framework, and drivers, such as the DOM Driver, are available as optional plugins for your app. This means it is simple to replace a driver with another one that has the same expected sinks and sources.
```

Comment:
Maybe: `Cycle.js is a very small framework, at its core it is just glue for drivers, such as [...] which are available [...].`?
Sounds a bit off ATM